src/test/java/com/echobox/api/linkedin/jsonmapper/DefaultJsonMapperTest.java

```diff
@@ -18,143 +18,41 @@
-    assertNotNull(locations.get(0).getAddress());
-    assertEquals("12 Swanee Dr", locations.get(0).getAddress().getFirstStreet());
-    assertNull(locations.get(0).getAddress().getSecondStreet());
-    assertEquals("Goddard", locations.get(0).getAddress().getCity());
-    assertEquals("Kansas", locations.get(0).getAddress().getState());
-    assertEquals("67051", locations.get(0).getAddress().getPostalCode());
-    assertEquals("us", locations.get(0).getAddress().getCountryCode());
-    assertEquals(new Integer(904), locations.get(0).getAddress().getRegionCode());
-    assertNotNull(locations.get(0).getContactInfo());
-    assertNull(locations.get(0).getContactInfo().getPhoneOne());
-    assertNull(locations.get(0).getContactInfo().getPhoneTwo());
-    assertNull(locations.get(0).getContactInfo().getFax());
-    assertNull(locations.get(0).getDescription());
-    
-    assertFalse(locations.get(1).isHeadquarters());
-    assertTrue(locations.get(1).isActive());
-    assertNotNull(locations.get(1).getAddress());
-    assertEquals("12345 Gigli Ct", locations.get(1).getAddress().getFirstStreet());
-    assertNull(locations.get(1).getAddress().getSecondStreet());
-    assertEquals("Los Altos Hills", locations.get(1).getAddress().getCity());
-    assertEquals("California", locations.get(1).getAddress().getState());
-    assertEquals("94857", locations.get(1).getAddress().getPostalCode());
-    assertEquals("us", locations.get(1).getAddress().getCountryCode());
-    assertEquals(new Integer(0), locations.get(1).getAddress().getRegionCode());
-    assertNotNull(locations.get(1).getContactInfo());
-    assertNull(locations.get(0).getContactInfo().getPhoneOne());
-    assertNull(locations.get(0).getContactInfo().getPhoneTwo());
-    assertNull(locations.get(0).getContactInfo().getFax());
-    assertNull(locations.get(1).getDescription());
-    
-    assertEquals("Some sort of description", company.getDescription());
-    assertNull(company.getStockExchangeRaw());
-    assertEquals(new Integer(2018), company.getFoundedYear());
-    assertNull(company.getEndYear());
-    assertEquals(new Integer(959), company.getNumFollowers());
-  }
-  
-  /**
-   * Test for the case LinkedIn may return a JSON array rather than the expected nested JSON array
-   * within an object can still convert to the correct object
+   * Ensure that the JSONMappingCompleted annotation is called
    */
   @Test
-  public void testToJavaListForJSONArray() {
-    String json = "{\"sausages\": [{\"code\":\"123\",\"name\":\"Bratwurst\"}]}";
-    LinkedInJsonMapper mapper = new LinkedInJsonMapper();
-    TestJSONMapper result = mapper.toJavaObject(json, TestJSONMapper.class);
-    assertEquals(1, result.getSausages().size());
-    assertEquals("123", result.getSausages().get(0).getCode());
-    assertEquals("Bratwurst", result.getSausages().get(0).getName());
+  public void testJSONMappingCompleted() {
+    LinkedInJsonMapperV1 mapper = new LinkedInJsonMapperV1();
```

Comment:
Should these not be instances of `DefautJSONMapper` in this test?

Reply:
Bad autorename :P